### ISSUE TITLE
add this overlay to gentoo api?

### ISSUE THREAD

**Issue by dm9pZCAq** [clang-musl-overlay/clang-musl-overlay#2]
this is needed to be able to add this overlay with just
```sh
eselect repository enable clang-musl # or other name which you come up with
```

---
* https://github.com/gentoo/api-gentoo-org
* https://wiki.gentoo.org/wiki/Project:Overlays/Overlays_guide#Requesting_adding_repository_hosted_elsewhere_.28the_new_way.29
* https://wiki.gentoo.org/wiki/Eselect/Repository

**Comment by leonardohn** [comment]
We will add to the official overlay list in the future, but we still have some issues to fix before doing it, like adding the patches from [our patchset](https://github.com/leonardohn/gentoo-patchset.git) as proper packages in the overlay.

**Comment by dante-robinson** [comment]
> 

Is the current musl/clang stage 3 image on the gentoo website built around this overlay? I am unsure where to comment about an issue using it I borked my previous install and started over with that stage 3 image everything works fine except x11-misc/compose-tables will not compile without using GCC or install llvm-conf from this overlay. The config should be included in the stage 3 if the 2 connected.

**Comment by capezotte** [comment]
@dante-robinson No, none of our packages are included there. We do provide stage 4s built from our repository in the Releases section of this repository, though (unfortunately they're fairly outdated).

**Comment by leonardohn** [comment]
@dante-robinson our stage3 project was created as a workaround for the lack of an official stage3 based on musl + clang about one year ago, and we have no relationship with the official Gentoo development team. I'm not sure in what development stage is the official stage3, but we have many patches to fix problems related to musl + clang that are not yet upstream, so one might find useful to add this overlay for the patches.

**Comment by dante-robinson** [comment]
> @dante-robinson our stage3 project was created as a workaround for the lack of an official stage3 based on musl + clang about one year ago, and we have no relationship with the official Gentoo development team. I'm not sure in what development stage is the official stage3, but we have many patches to fix problems related to musl + clang that are not yet upstream, so one might find useful to add this overlay for the patches.

Should I report any issues with compiling applications here then?

**Comment by leonardohn** [comment]
@dante-robinson we usually recommend people to report problems at the official channels directly to the software developers, as they have the expertise to find solutions for their software faster than we do. However, as it might take a while for a fix to be upstreamed, we also accept reports and pull requests in our patchset, so feel free to report any problems or send new patches :)
